controller/system_rollout_controller_test.go

```diff
@@ -208,7 +208,7 @@ func (s *TestSuite) TestSystemRollout(c *C) {
 					RoleRef: rbacv1.RoleRef{Name: TestClusterRoleName},
 				},
 			},
-			expectClusterRoleBindings: map[SystemRolloutCRName]*rbacv1.ClusterRoleBinding{
+			backupClusterRoleBindings: map[SystemRolloutCRName]*rbacv1.ClusterRoleBinding{
```

Comment:
I think you can do this way. WDYT?

`expectedRestoredClusterRoleBinding`, the same for others.

Reply:
I'm trying to un-confuse the case for `system rollout Settings ignored`:
So here is 
```
                        existSettings: map[SystemRolloutCRName]*longhorn.Setting{
				SystemRolloutCRName(types.SettingNameConcurrentReplicaRebuildPerNodeLimit): {Value: "4"},
				SystemRolloutCRName(types.SettingNameConcurrentBackupRestorePerNodeLimit):  {Value: "5"},
			},
			backupSettings: map[SystemRolloutCRName]*longhorn.Setting{
				SystemRolloutCRName(types.SettingNameConcurrentReplicaRebuildPerNodeLimit): {Value: "6"},
				SystemRolloutCRName(types.SettingNameConcurrentBackupRestorePerNodeLimit):  {Value: "7"},
			},
```
vs
```
                        existSettings: map[SystemRolloutCRName]*longhorn.Setting{
				SystemRolloutCRName(types.SettingNameConcurrentReplicaRebuildPerNodeLimit): {Value: "4"},
				SystemRolloutCRName(types.SettingNameConcurrentBackupRestorePerNodeLimit):  {Value: "5"},
			},
			expectedRestoredSettings: map[SystemRolloutCRName]*longhorn.Setting{
				SystemRolloutCRName(types.SettingNameConcurrentReplicaRebuildPerNodeLimit): {Value: "6"},
				SystemRolloutCRName(types.SettingNameConcurrentBackupRestorePerNodeLimit):  {Value: "7"},
			},
```

Reply:
Update to:
- **backup**Foobar
- **exist**Foobar
- **expectRestored**Foobar